ibc/host.go

```diff
@@ -1,8 +1,14 @@
+// TECHDEBT(#854): The host should be a submodule with access to the bus, this allows for the
```

Comment:
@dylanlott I believe this is why we need to register the tree submodule rather than just set the bus. So its accessible from here

Reply:
For posterity: TreeStore registration on the bus is added by #861